Clinical-Genomics/scout
# Load SVs for demo case 643595

I've noticed that SVs for the clone case 643595 are not loaded. They are useful, so I'm introducing them with this PR!

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Locally, main branch, load case 643595: `scout --demo load scout/demo/643595.config.yaml`
2. Go to case and make sure it has no SVs loaded
3. Remove case: `scout --demo delete case -i cust000 -c internal_id_2`
4. Switch to this branch and repeat 1.
5. Now SVs should be loaded

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
